src/main/java/Deadline.java

```diff
@@ -0,0 +1,13 @@
+public class Deadline extends Task {
```

Comment:
I believe organising your Deadline, Todo, Event and task into a single package will help with the structure of your files. Also helps as you implement more classes.